### ISSUE TITLE
[CS 1.6] [Feature Request] Delay between Radio Messages

### ISSUE THREAD

**Issue by Maxi605** [ValveSoftware/halflife#2616]
Add client-side delay for the radio messages, after sending the first radio message block the next radio message for 3 seconds to avoid player spamming radio messages, this could be problematic when you called something out with the radio and you immediatly need to give another message, in that case a server-side would be better to be able to decide on a good time limit in between messages.

**Comment by BUXPb** [comment]
Not. For this you have to use the "votekick".

The delay between radio messages will significantly disrupt the gameplay, especially if you need to send several radio messages in a row and they must be delivered immediately. A good solution to the problem is to extend the functionality of the "mute" of a particular player, so that it would be possible to block not only his voice messages, but also the radio commands coming from him, as well as chat messages

**Comment by GiovaniFerraroTrivelli** [comment]
> The delay between radio messages will significantly disrupt the gameplay, especially if you need to send several radio messages in a row and they must be delivered immediately.

Yes, but adding a cvar is an option too. I think this is a good idea.
